src/System.Private.CoreLib/src/System/Reflection/MdFieldInfo.cs

```diff
@@ -29,11 +30,11 @@ internal sealed unsafe class MdFieldInfo : RuntimeFieldInfo
         #endregion
 
         #region Internal Members
-        internal override bool CacheEquals(object o)
+        internal override bool CacheEquals(object? o)
         {
-            MdFieldInfo m = o as MdFieldInfo;
+            MdFieldInfo? m = o as MdFieldInfo;
 
-            if ((object)m == null)
```

Comment:
Why did you delete this?  It's a semantic change, as FieldInfo provides an `operator==`.  While it's probably unlikely to matter, this should really either be changed to `if (m is null)` or `if ((object?)m == null)`, both of which will result in the same IL as before, whereas this change won't.

Reply:
i was changing them to if (m is null) but seems here forgot to change `==` to `is`

Reply:
this is already fixed